tests/unit/fem/test_estimator.cpp

```diff
@@ -244,7 +248,7 @@ TEST_CASE("Least-squares ZZ estimator on 3D NCMesh", "[NCMesh]")
       {
          REQUIRE(local_errors(i) < 1e-10);
       }
-      REQUIRE(estimator.GetTotalError() < 1e-10);;
+      REQUIRE(estimator.GetTotalError() < 1e-10);
```

Comment:
`REQUIRE` is for establishing a predicate of whether a test can meaningfully continue, to establish a test return, use `CHECK`.